### DISCUSSION TITLE
Can visual studio get faster by only using intellisense and the debug button

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/roslyn#66175] [General]
Visual studio code is very fast, but visual studio is convenient because I can add project reference edit project property in the UI, a visual studio that's light weight really helps

**Comment by Xyncgas** [comment]
Whenever I launch visual studio, my laptop goes hot and noisy, a lot is going on than a text editor with useful function (on demand) and intellisense (dynamic)

**Comment by CyrusNajmabadi** [comment]
@Xyncgas this is the repo for roslyn, not vs. Please file vs feedback using the Report feature it has. Please also include a perf trace so the relevant teams can look into it.
